contracts/L1/gateway/L1LPTGateway.sol

```diff
@@ -88,6 +88,7 @@ contract L1LPTGateway is IL1LPTGateway, ControlledGateway, L1ArbitrumMessenger {
         bytes calldata _data
     ) external payable override whenNotPaused returns (bytes memory) {
         require(_l1Token == l1Lpt, "TOKEN_NOT_LPT");
+        require(_amount > 0, "INVALID_ZERO_AMOUNT");
```

Comment:
only added the check in outbound transfer of both gateways and not inbound as it will be handled by the counterpart